added chronicles: battle for greece dlc resources

mardaravicius opened the pull request:
new resources from the new effects found in the a.g.e.
works if changed via xs scripts on the current version

Divy1211 commented on docs/general/resources/res_desc.json:
```diff
@@ -3233,5 +3233,49 @@
         "note": ""
+    },
+    "502": {
+        "name": "Chopping food productivity",
```
Make title case for name "Chopping Food Productivity"

Divy1211 commented on docs/general/resources/res_desc.json:
```diff
@@ -3233,5 +3233,49 @@
+        "note": ""
+    },
+    "503": {
+        "name": "Trade wood percentage",
```
Make title case for name "Trade Wood Percentage"

Divy1211 commented on docs/general/resources/res_desc.json:
```diff
@@ -3233,5 +3233,49 @@
+        },
+        "note": "Setting it to 0.5 will return half the amount in gold and half in wood. Seems to be working for trade cogs only. Setting it to a value >= 1 will make it so no trade resources are returned"
+    },
+    "507": {
```
Make name consistent with other resources: "Unknown Resource 507"

Divy1211 commented on docs/general/resources/res_desc.json:
```diff
@@ -3233,5 +3233,49 @@
+        },
+        "note": "Unknown, but probably something related to Athenian policies"
+    },
+    "521": {
```
Are resources 508-520 unused? If so, could you copy the format from one of the unused resources above and create "Unused Resource 509" to "Unused Resource 520" for completeness? If their purposes are unknown, then use "Unknown" instead of "Unused" in the name

mardaravicius replied:
since the dlc patch i think they increased the resource limit by 100. 
and these are the resources i found in a.g.e. effects.

Divy1211 commented on docs/general/resources/res_desc.json:
```diff
@@ -3233,5 +3233,49 @@
+        },
+        "note": "The amount of gold obtained from owning `n` number of Castles is given by `n * castle_gold_productivity * 0.33333`"
+    },
+    "551": {
```
Same for resources 522-550 as 508-520

Divy1211 commented on docs/general/resources/res_desc.json:
```diff
@@ -3233,5 +3233,49 @@
+            "1": " After Tyranny researched"
+        },
+        "note": ""
     }
```
I believe we have usable resources up to 600 now, same as resources 508-520 and 522-550

mardaravicius commented:
added all gap resources and called unknown. since I don't if they are actually in use.
also added more resources i found in a.g.e. units (501, 506, 509, 520, 550).
clarified 551 after testing it more

Divy1211 commented:
thanks for the updates!
